src/gui/torrentcontentmodel.cpp

```diff
@@ -351,6 +351,12 @@ QVariant TorrentContentModel::data(const QModelIndex &index, int role) const
                 return Qt::PartiallyChecked;
             return Qt::Checked;
         }
+    case Qt::TextAlignmentRole: {
+        if ((index.column() == TorrentContentModelItem::COL_SIZE)
+            || (index.column() == TorrentContentModelItem::COL_REMAINING))
```

Comment:
If there is going to add another column then we need to use `switch ()`.

Reply:
> If there is going to add another column then we need to use switch ().

yes leave it for another time then 😅😛